src/components/landscape-sidebar/conservation-efforts-widget/conservation-efforts-widget.js

```diff
@@ -72,7 +72,7 @@ const ConservationEffortsWidget = (props) => {
   useEffect(() => {
     if (terrestrialCellData && queryParams) {
       setConservationEfforts({ data: null, loading: true });
-      queryParams.where = `CELL_ID IN (${terrestrialCellData.map(i => i.CELL_ID).join(', ')})`;
+      queryParams.where = `ID IN (${terrestrialCellData.map(i => i.ID).join(', ')})`;
```

Comment:
If you do not filter out the marine grid cells on the selector when querying the layer on coastal zones (with terrestrial and marine cells) will not return any data (because the resulting array of the mapping will have some `undefined` elements.